sbom/cve-bin-tool-py3.9.spdx.vex.json

```diff
@@ -0,0 +1,70 @@
+{
+  "@context": "https://openvex.dev/ns/v0.2.0",
+  "@id": "https://openvex.dev/docs/public/vex-d7971dad596bf8f0395a9da7c1c0a039ad75e431414c45d09dbcc3c2d4f9c989",
+  "author": "Unknown Author",
+  "timestamp": "2024-10-04T15:31:21.082828+02:00",
+  "last_updated": "2024-10-04T15:31:21.194452+02:00",
+  "version": 5,
+  "statements": [
+    {
+      "vulnerability": {
+        "name": "CVE-2024-35195"
+      },
+      "timestamp": "2024-10-04T15:31:21.082828+02:00",
+      "products": [
+        {
+          "@id": "pkg:pypi/requests@2.31.0"
+        }
+      ],
+      "status": "under_investigation"
+    },
+    {
+      "vulnerability": {
+        "name": "CVE-2024-37891"
+      },
+      "timestamp": "2024-10-04T15:31:21.112094+02:00",
+      "products": [
+        {
+          "@id": "pkg:pypi/urllib3@2.2.1"
+        }
+      ],
+      "status": "under_investigation"
+    },
+    {
+      "vulnerability": {
+        "name": "CVE-2024-39689"
+      },
+      "timestamp": "2024-10-04T15:31:21.13969+02:00",
+      "products": [
+        {
+          "@id": "pkg:pypi/certifi@2024.2.2"
+        }
+      ],
+      "status": "under_investigation"
+    },
+    {
+      "vulnerability": {
+        "name": "CVE-2024-42367"
+      },
+      "timestamp": "2024-10-04T15:31:21.166505+02:00",
+      "products": [
+        {
+          "@id": "pkg:pypi/aiohttp@3.9.5"
```

Comment:
requirements.txt currently says `aiohttp[speedups]>=3.9.4` so we may want to bump the minimum here.  I'd rather do that than maintain a vex for this one.